Essentials/src/main/java/com/earth2me/essentials/MetaItemStack.java

```diff
@@ -45,6 +42,8 @@
 import static com.earth2me.essentials.I18n.tl;
 
 public class MetaItemStack {
+    private static final String DEFAULT_BOOK_AUTHOR = Bukkit.getConsoleSender().getName();
```

Comment:
Not sure I like this... definitely more of an opinionated thing but I'd much rather it default to the player requesting/spawning the book, rather than console. Either that or make this configurable.